instructions/MIGRATION_0.2.md

```
@@ -0,0 +1,103 @@
+`CurrentUser → AuthUser` unification (delivered as part of `reinhardt-auth`).
+
+> **Status:** This file is filled in incrementally as each per-crate PR
+> lands. Empty sections below are placeholders for upcoming PRs.
+
+---
+
+## Quick removal index
+
+| Crate | PR | Status | Section |
+|---|---|---|---|
+| `reinhardt-core` | TBD | ⏳ pending | [reinhardt-core](#reinhardt-core) |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)